packages/react-search-ui/README.md

```diff
@@ -600,18 +602,18 @@ ex.
+| `addFilter`         | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                                      |        | Add a filter in addition to current filters values                         |
+| `setFilter`         | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                                      |        | Set a filter value, replacing current filter values                        |
+| `removeFilter`      | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                                      |        | Remove a single filter value                                               |
+| `reset`             |                                                                                                                                                                                            |        | Reset state to initial search state                                        |
+| `clearFilters`      |                                                                                                                                                                                            |        | Clear all filters                                                          |
+| `setCurrent`        | Integer                                                                                                                                                                                    |        | Update the current page number, used for paging                            |
+| `setResultsPerPage` | Integer                                                                                                                                                                                    |        |                                                                            |
+| `setSearchTerm`     | `searchTerm` String<br/> `options` Object<br/>`options.refresh` Boolean - Refresh search results on update. Default: `true`.<br/>`debounce` Number - Length to debounce if using `refresh` |        |                                                                            |
+| `setSort`           | `sortField` String - field to sort on<br/>`sortDirection` String - "asc" or "desc"                                                                                                         |        |                                                                            |
+| `trackClickThrough` | `documentId` String - The document ID associated with the result that was clicked<br/>`tag` - Array[String] Optional tags which can be used to categorize this click event                 |        | Report a clickthrough event. which is when a user clicks on a result link. |
```

Comment:
Last part of this sentence is weird. Not sure that period belongs there. "...event. which..."

Reply:
Done.